### PR TITLE
Lw/6478 migrate to jest poc

### PR DESCRIPTION
**Pull request by johnnieskywalker**
# [Title](https://shortcutLink)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>
- item 1
  
## How to test 🧪
  <Please explain how to test your changes>
- step 1 ...

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+const customJestConfig = {
+  setupFilesAfterEnv: ['<rootDir>/jest.setup.js'],
+  moduleNameMapper: {
+    // Handle module aliases (this will be automatically configured for you soon)
+    '^@/components/(.*)$': '<rootDir>/components/$1',
+
+    '^@/pages/(.*)$': '<rootDir>/pages/$1',
+  },
+  testEnvironment: 'jest-environment-jsdom',
+}
```
Comment (johnnieskywalker) [inline jest.config.js:18]: This commit is not breaking mocha, all mocha test work and jest tests work like this 
Test Suites: 39 failed, 5 passed, 44 total
Tests:       13 failed, 36 passed, 49 total
Snapshots:   0 total
Time:        5.483 s

image

Comment (johnnieskywalker) [reply jest.config.js:18]: features/test
image

Comment (johnnieskywalker) [reply jest.config.js:18]: helpers
image

image

**Comment by marcinciarka** [comment]
Closing this old PR. Please ping me for details if needed.

**Pull request closed without merging**